crates/nargo/src/cli/mod.rs

```diff
@@ -78,6 +76,8 @@ pub fn start_cli() {
 
 // helper function which tests noir programs by trying to generate a proof and verify it
 pub fn prove_and_verify(proof_name: &str, prg_dir: &Path, show_ssa: bool) -> bool {
+    use tempdir::TempDir;
```

Comment:
You put it here to signify that its not needed in the module namespace and only the function namespace?

Reply:
Yeah, we only use this dependency for the integration tests so I placed it here to avoid it leaking into the "real" code.

Reply:
sounds good ser